examples/player.html

```diff
@@ -126,10 +126,11 @@ <h3 class="text-center">WebRTC Client Player Example</h3>
             // import { createApp } from 'https://cdn.jsdelivr.net/npm/vue@3/dist/vue.esm-browser.js';
             // production version, optimized for size and speed
             import { createApp } from 'https://cdn.jsdelivr.net/npm/vue@3/dist/vue.esm-browser.prod.js';
-            console.log('webrtc-client version:', VERSION);
-            console.log('web-utils version:', utils.VERSION);
 
-            const { Util, NetAddress } = utils;
+            const { Util, NetAddress, log } = utils;
+
+            log('webrtc-client version:', VERSION).info();
+            log('web-utils version:', utils.VERSION).info();
```

Comment:
It's a trick that we have found with @thomasjammet to not import `web-utils` in addition from `webrtc-client` since than this last use and imports web-utils as well.

Be careful not to confuse **Util** and **web-utils**,
here **utils** = **web-utils**, and inside you have multiple classes like [BinaryReader, BinaryWriter, Connect, EventEmitter, ..., and Util](https://github.com/CeeblueTV/web-utils/tree/main/src).

So rather to import `web-utils` one time again:
`import { Util, NetAddress, log } from "../dist/web-utils.min.js";`
we import the one inside `webrtc-client`
`import { ..., utils } from "../dist/webrtc-client.min.js";`
And use it with the following trick to extract classes inside web-utils like if was a normal import:
`const { Util, NetAddress, log } = utils;`

Reply:
Is it possible to use use something like webrtc-client.VERSION to make lines approximately the same length?

Reply:
> ...to make lines approximately the same length

maniac. but the pb is here it kills what we want to illustrate.

Indeed we can by changing the classical import:
`import { Player, HTTPConnector, WSController, VERSION, utils } from "../dist/webrtc-client.min.js";`
by
`import * as webrtcClient from "../dist/webrtc-client.min.js";`
And after prefix all the class by `webrtcClient.WSController` etc.. what is less conventional, where as the goal of `player.html` is precisely to show how use it correctly.

So at least that it prevents you to sleep, I request that you accept this line with different width :-)